devices/elko/thermostat_16_a.json

```diff
@@ -0,0 +1,356 @@
+            },
+            {
+               "name":"attr/swversion"
+            },
+            {
+               "name":"attr/type"
+            },
+            {
+               "name":"attr/uniqueid"
+            },
+            {
+               "name":"config/offset",
+               "default":0
+            },
+            {
+               "name":"config/on"
+            },
+            {
+               "name":"config/reachable"
+            },
+            {
+               "name":"state/lastupdated"
+            },
+            {
+               "name":"state/temperature"
+            }
+         ]
+      },
+      {
+         "type":"$TYPE_TEMPERATURE_SENSOR",
+         "restapi":"/sensors",
+         "uuid":[
+            "$address.ext",
+            "0x03",
+            "0x0402"
+         ],
+         "items":[
+            {
+               "name":"attr/id"
+            },
+            {
+               "name":"attr/lastannounced"
+            },
+            {
+               "name":"attr/lastseen"
+            },
+            {
+               "name":"attr/manufacturername"
+            },
+            {
+               "name":"attr/modelid"
+            },
+            {
+               "name":"attr/name"
+            },
+            {
+               "name":"attr/swversion"
+            },
+            {
+               "name":"attr/type"
+            },
+            {
+               "name":"attr/uniqueid"
+            },
+            {
+               "name":"config/offset",
+               "default":0
+            },
+            {
+               "name":"config/on"
+            },
+            {
+               "name":"config/reachable"
+            },
+            {
+               "name":"state/lastupdated"
+            },
+            {
+               "name":"state/temperature"
+            }
+         ]
+      },
+    {
+      "type": "$TYPE_CONSUMPTION_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x05",
+        "0x0702"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0000",
+          "0x0702"
+        ]
+      },
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/consumption",
+          "refresh.interval": 500,
+          "default": 0
```

Comment:
Could you please remove `"default": 0` and do not forget to also remove the comma in the line above :)